platform/ui/src/components/measurementTable/MeasurementTable.styl

```diff
@@ -84,6 +84,106 @@
         position: relative
         top: 2px
 
+  &.lesionTracker
+    .displayTexts
+      display: flex
+
+      .measurementDisplayText
+        flex: 1
+
+    .measurementTableHeader
+      display: flex
+      position: relative
+      padding-left: 38px
+
+      .warning-status
+        left: 0
+        top: 3px
+        cursor: pointer
+        width: 40px
+        height: 100%
+        display: flex
+        align-items: center
+
+        .warning-border
+          padding: 2px 3px
+          width: 30px
+          height: 30px
+
+        svg
+          width: 20px
+          height: 18px
+
+      .measurementTableHeaderItem
+        position: relative
+        flex: 1
+
+        .caseProgressContainer
+          position: absolute
+          right: 4px
+          top: 0
+
+          .caseProgress
+            -webkit-transition: all 0.3 ease
+            -moz-transition: all 0.3 ease
+            -ms-transition: all 0.3 ease
+            -o-transition: all 0.3 ease
+            transition: all 0.3 ease
+
+            .radialProgress
+              display: inline-block
+              float: left
+              -webkit-border-radius: 100%
+              -moz-border-radius: 100%
+              -ms-border-radius: 100%
+              -o-border-radius: 100%
+              border-radius: 100%
+              height: 26px
+              margin-top: 3px
+              margin-right: 4px
+              position: relative
+              -webkit-box-shadow: 0 0 1em #000
+              -moz-box-shadow: 0 0 1em #000
+              -ms-box-shadow: 0 0 1em #000
+              -o-box-shadow: 0 0 1em #000
+              -webkit-box-shadow: 0 0 1em #000
+              -moz-box-shadow: 0 0 1em #000
+              -ms-box-shadow: 0 0 1em #000
+              -o-box-shadow: 0 0 1em #000
+              box-shadow: 0 0 1em #000
+              width: 26px
+
+              #svg
+                circle
+                  stroke: #3c5d80
+                  stroke-width: 3px
+                  -webkit-transition: stroke-dashoffset 1s linear
+                  -moz-transition: stroke-dashoffset 1s linear
+                  -ms-transition: stroke-dashoffset 1s linear
+                  -o-transition: stroke-dashoffset 1s linear
+                  -webkit-transition: stroke-dashoffset 1s linear
+                  -moz-transition: stroke-dashoffset 1s linear
+                  -ms-transition: stroke-dashoffset 1s linear
+                  -o-transition: stroke-dashoffset 1s linear
+                  transition: stroke-dashoffset 1s linear
```

Comment:
No vendor prefixes. If we have issues on those platforms, something like PostCSS should be generating vendor prefixes based on our build target on our behalf.

Reply:
Done, thanks.